src/char/char.c

```diff
@@ -2414,7 +2414,7 @@ static int char_parse_fromlogin_connection_state(int fd)
 	case 1: // Invalid username/password
 		ShowError("Can not connect to login-server.\n");
 		ShowError("The server communication passwords (default s1/p1) are probably invalid.\n");
-		ShowError("Also, please make sure your login db has the correct communication username/passwords and the gender of the account is S.\n");
+		ShowError("Also, please make sure your login db has the correct communication username/passwords, the gender of the account is S and its id is between 0 and (MAX_SERVERS - 1).\n");
```

Comment:
"0" is not a typo, the check on `login.c` really checks for `>= 0` (see below), and testing with an account id = 0, it does seem to work. I always thought 0 index was somehow reserved, but looks like it isn't.

Whether we want to keep this or not... I am in favor of changing the minimum ID to 1.

Reply:
0 seems indeed valid, even though I don't believe I've ever used it before. It makes sense it was implemented this way since we use it as array index for `login->dbs->server[]` (and also `login->dbs->api_server[]`), the 'invalid' value we use is `-1`.

I don't have a compelling reason to keep it this way or to change it. If you find it confusing, feel free to change how it works (in a separate PR)!

Reply:
Ah, I think I made a confusion between `fd = 0` and `account_id = 0`. `fd = 0` is reserved, but this only affects sockets and doesn't relate to account id. so it is fine